torch/quantization/fx/lower_to_native_backend.py

```diff
@@ -0,0 +1,8 @@
+import torch
+
+def lower_to_native_backend(model: torch.fx.GraphModule):
```

Comment:
should the type be `nn.Module`, and we trace again?

should we create `lower_to_fbgemm` and `lower_to_qnnpack`, even if the code is the same for now?  They may diverge in the future, and this way the API is more future proof.

Reply:
Sure, we can trace again.

Yeah I can split the function into 2.

Reply:
actually if we trace again we might need to carry over configurations from quantization, which is a little bit cumbersome, can we use this as an internal function and just accept torch.fx.GraphModule instead?

Reply:
I think it is ok to have this be an internal function as users interface with this only via convert_fx.